docs/Homebrew-homebrew-cask-Maintainer-Guide.md

```diff
@@ -0,0 +1,61 @@
+# Homebrew/homebrew-cask Maintainer Guide
+
+This guide is intended to help maintainers effectively maintain the cask repositories.
+It is meant to be used in conjunction with the more generic [Maintainer Guidelines](Maintainer-Guidelines.md).
+
+This guide applies to all four of the cask repositories:
+
+- [Homebrew/homebrew-cask](https://github.com/Homebrew/homebrew-cask): The main cask repository
+- [Homebrew/homebrew-cask-drivers](https://github.com/Homebrew/homebrew-cask-drivers): Casks of drivers
+- [Homebrew/homebrew-cask-fonts](https://github.com/Homebrew/homebrew-cask-fonts): Casks of fonts
+- [Homebrew/homebrew-cask-versions](https://github.com/Homebrew/homebrew-cask-versions): Alternate versions of Casks
+
+## Common Situations
+
+Here is a list of the most common situations that arise in cask PRs and how to handle them:
+
+- The `version` and `sha256` both change (keeping the same format): Merge.
+- Only the `sha256` changes: Merge unless the version needs to be updated as well.
+  It’s not uncommon for upstream vendors to update versions in-place.
+- `livecheck` is updated: Use your best judgement and try to make sure that the changes
+  follow the [`livecheck` guidelines](Brew-Livecheck.md).
+- Only the `version` changes or the `version` format changes: Use your best judgement and
+  merge if it seems correct (this is relatively rare).
+- Other changes (including adding new casks): Use the [Cask Cookbook](Cask-Cookbook.md) to determine what's correct.
+
+If in doubt, ask another cask maintainer on GitHub or Slack.
+
+Note that unlike formulae, casks do not consider the `sha256` stanza to be a meaningful security measure
+as maintainers cannot realistically check them for authenticity. Casks download from upstream; if a malicious
+actor compromised a URL, they could just as well compromise a version and make it look like an update.
+
+## Merging
+
+### Approvals
+
+Most PRs in the cask repositories are simple version bumps that don't necessarily need
+another maintainer's approval. However, GitHub will not allow a PR to be merged without
+at least one approving review. To bypass this requirement if necessary, a maintainer
+can self-approve one of their PRs using the `self-approve` GitHub Actions workflow to
+satisfy this requirement. To trigger a self-approval, navigate to the
+["Self-approve a Pull Request" section of the Actions tab](https://github.com/Homebrew/homebrew-cask/actions/workflows/self-approve.yml),
+click on "Run workflow", enter the PR number and click "Run workflow".
```

Comment:
How does this differ from Homebrew/homebrew-core? Is self-approval required there too?

Reply:
There is no self-approve workflow in homebrew/core at the moment. The reason is that members of the `@Homebrew/core` team have admin access to the homebrew/core repo. This means that they can override the branch protection rules and merge without approval if desired.

Members of the `@Homebrew/cask` team, though, only have maintain access to the cask repos, so they can't override the branch protection rule to merge if there are no approvals. So, to get around it, there's the self-approval workflow to have @BrewTestBot approve the PR, fulfilling the requriements.

(To be fair I haven't really tested this so it's a bit of speculation and it's possible that I'm wrong and individuals with maintain access _can_ override. I'm pretty sure that's not the case, though)

Reply:
@Rylan12 I'd be tempted to make these behaviours consistent. @Homebrew/cask any objections to getting rid of this self-approve workflow in favour of cask maintainers being administrators?

Reply:
@MikeMcQuaid I’m generally in favour of consistent processes. It reduces cognitive load, confusion and errors and makes onboarding easier.

Reply:
> any objections to getting rid of this self-approve workflow

Not only don’t I object, I _welcome_ getting rid of it!

Reply:
In that case, I'll remove the section from the documentation.

Reply:
@vitorgalvao Done!